metadata/readme/0061.md

```
@@ -0,0 +1,54 @@
+# sqlmap
+
+Installation
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)